rules-topics/15_smithing.md

```diff
@@ -0,0 +1,99 @@
+#### Reinforcement Kit
+>This kit helps prepare the user for imminent destruction by providing them with several small tools that when quickly applied can help prevent an item from being destroyed. This can be prepared with 1 Minute of roleplay. It grants the user a single use of Resist Destruction, that lasts until Rest. An individual may only have one Reinforcement Kit active on it at time. This item is consumed on use. This does not take a defense slot.
+#### Weighted Ammunition
+   >This specially designed item is made from very dense materials, and is made to blow past enemy defenses to really hit them where it hurts. This ammunition can either be fired from a Ranged Weapon or used as a Thrown Weapon. While it carries a heavy impact, the design means that its unwieldy nature makes it difficult to use with precision. Using this ammunition will deal 5 Normal Vital damage, this damage call cannot be modified any means. This item is consumed on use.
+
+### Tier VIII
+
+#### Expandable Weapon
+> A marvel of intricate crafting, this weapon modification is designed to allow the wielder the ultimate flexibility in controlling their reach. With 60 seconds of roleplay, the wielder may change the length of this weapon, from up to a Two-Handed weapon down to a Short length one-handed weapon. This allows the wielder to use this item with a Short, Long, or Two-Handed weapon rep interchangably.
+#### Recycling Kit
+> This kit is designed to assist in helping a Recycler be more efficient with their reclamation of materials. When a kit is made, a single Crafting Type is chosen from Alchemy, Smithing or Rune Carving. When the user recycles an item from that type, they may expend a charge from this kit to recover 1/2 of the ranks in materials used to construct it, rounded down to a minimum of one, instead of the usual 1/3. In the case of Recycling an Artificing item, the user only benefits from this increase for the materials of the chosen type, though multiple kits may be used in tandem. This item has 3 charges.
+#### Trickleblade Modification
+> This modification can be applied to any weapon, adding a resevoir and specially treated channels to the weapon that allow Toxins to flow along the edge of the blade. The wielder of this blade may apply Ingestible Toxins to this as if they were Weapon Coatings.
+
+
+
+### Tier IX
+
+#### Armor Repair Kit
+   >This small kit has everything an individual needs to help fix all dents, scratches, tears, breaks, and other mishaps! This kit will let the user, on a three count, refit an armor to its full value upon use. It has no effect on destroyed armor. This functions as per Rapid Refit. This item is consumed on use.
```

Comment:
Would you mind updating `three count` to `3 second count` here?  It's possible we missed this previously when we tried to clarify them all as seconds.